modules/domain-templates-knative-services.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative-services.adoc
+
+[id="domain-templates-knative-services_{context}"]
+= Defining a template for the configured domain
+
+.Procedure
+. Define a `domainTemplate`.
```

Comment:
This note isn't specific to the line, but applies to the entire file. Because you use spacing for indentation, the ordered list breaks. You have two steps labeled as step 1.

I'd strongly recommend using `+` to connect steps in a procedure, and wrapping source blocks in `----` to explicitly mark their beginning and ending. This should also help maintenance should these sections need to change in the future.